pkgs/applications/networking/instant-messengers/teamspeak/client.nix

```diff
@@ -1,19 +1,19 @@
 { lib, stdenv, fetchurl, makeWrapper, makeDesktopItem, zlib, glib, libpng, freetype, openssl
 , xorg, fontconfig, qtbase, qtwebengine, qtwebchannel, qtsvg, qtwebsockets, xkeyboard_config
-, alsa-lib, libpulseaudio ? null, libredirect, quazip, which, unzip, llvmPackages_10, writeShellScriptBin
+, alsa-lib, libpulseaudio ? null, libredirect, quazip, which, unzip, perl, llvmPackages
```

Comment:
It might be worth keeping this pined as `llvmPackages_16` given that dependency has broken it before. But I rarely use team speak any more (because of discord) so either look good to me.

Reply:
We now know what causes this. When it happens again, I'll know how to fix it.